sdk/core/core-client-paging-rest/package.json

```diff
@@ -9,6 +9,9 @@
   "browser": {
     "./dist-esm/src/url.js": "./dist-esm/src/url.browser.js"
   },
+  "react-native": {
+    "./dist/index.js": "./dist-esm/src/index.js"
+  },
```

Comment:
@joheredi is this package still a dep in the rest clients? if not, we could probably skip updating it. Same applies for the lro one.